.gitignore

```diff
@@ -21,6 +21,10 @@ build/_vendor/pkg
 *~
 .project
 .settings
+.git
+
+# hardhat private keys
+hardhat.config.js
```

Comment:
Is it necessary to have hardhat keys in the go-quai folder? Or should those stay in that repository. I'm hesitant to add git ignore dependencies for specific other projects